Release 40 (#40)
Azure/API-Management: changelogs/api-management-service.md

# Azure API Management service changelog

# Release - API Management service: October 2023

### Highlights

1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2.(https://learn.microsoft.com/en-us/azure/api-management/v2-service-tiers-overview)

### New features, fixes and improvements

1. We have added a feature flag for exporting API securitywhen exporting customer API to swagger.

### Self-hosted gateway container image releases

1. [2.3.5](https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/Container-v2.3.5)

### Self-hosted gateway Helm chart releases

1.[1.7.5](https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/v1.7.5)

## Release - API Management service: September 2023

### New features, fixes, and improvements

1. We have introduced new Azure Advisor recommendations

>     -  Use Azure AD authentication instead of gateway token for Configuration API

Reviewer: ```suggestion
    - Use Azure AD authentication instead of gateway token for Configuration API
```

# Azure API Management service changelog

# Release - API Management service: October 2023

### Highlights

> 1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2.(https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview)

Reviewer: ```suggestion
1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2. ([Docs](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview))
```

### New features, fixes and improvements

1. We have added a feature flag for exporting API securitywhen exporting customer API to swagger.

> ### Self-hosted gateway container image releases

Reviewer: ```suggestion
### Developer portal releases

No releases.

### Self-hosted gateway container image releases
```

# Azure API Management service changelog

# Release - API Management service: October 2023

### Highlights

1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2. ([Docs](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview))

### New features, fixes and improvements

> 1. We have added a feature flag for exporting API securitywhen exporting customer API to swagger.

Reviewer 2: @tomkerkhove, it looks like there are two issues here:

1) _securitywhen_ should be _security when_
2) _swagger_ should be _Swagger_

Reviewer 3: > swagger should be Swagger

I'd argue that it should even be OpenAPI specification. Also, it's unclear from the notes how customers can use this feature flag - it would be good to explain it or point to the docs.

Reviewer 2: @mikebudzynski, I agree with you. We should be using _OpenAPI Specification_ anywhere we use _Swagger_ going forward as we want to reference the spec, not the product implementing it. I think we have outgrown historic reasons to use _Swagger_ now.

# Azure API Management service changelog

# Release -API Management service: October 2023

### Highlights

> 1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2.(https://learn.microsoft.com/en-us/azure/api-management/v2-service-tiers-overview)

Reviewer: ```suggestion
1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2.(https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview)
```

> ### New features, fixes and improvements
> 
> 1. We have added a feature flag for exporting API securitywhen exporting customer API to swagger.

Reviewer: ```suggestion
### New features

None.

### Fixes and improvements

1. We have added a feature flag for exporting API security when exporting customer API to swagger.
```

Reviewer: Aligned as per discussion, but is this a customer-facing feature flag or not? If not, then we should remove it

Reviewer 2: @tomkerkhove, _swagger_ is a proper noun and should be capitalized. It's also capitalized [here](https://swagger.io/solutions/getting-started-with-oas/).

>    

Reviewer: Another nit
```suggestion
```

### Self-hosted gateway container image releases

1. [2.3.5](https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/Container-v2.3.5)

### Self-hosted gateway Helm chart releases

1.[1.7.5](https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/v1.7.5)

> 
> 

Reviewer: Massiven it
```suggestion
```

### New features, fixes, and improvements

1. We have introduced new Azure Advisor recommendations

>     -  Use Azure AD authentication instead of gateway token for Configuration API

Reviewer: Fixed

    - Identity self-hosted gateways using gateway tokens that will expire in < 7 days
    - Identity validate-jwt policy usage with insecure key size

> 2. We have fixed an issue where the OpenApi would fail if the parameter names where empty.

Reviewer 2: Markdown lets you use `1.` for every numbered item and will automatically number correctly for you. That means you can more easily make changes to the numbered list without having to update the numbering itself.
